test/with-default-focused-input/AutosuggestApp.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import sinon from 'sinon';
+import Autosuggest from '../../src/Autosuggest';
+import languages from '../plain-list/languages';
+import { escapeRegexCharacters } from '../../demo/src/components/utils/utils.js';
+
+const getMatchingLanguages = value => {
+  const escapedValue = escapeRegexCharacters(value.trim());
+  const regex = new RegExp('^' + escapedValue, 'i');
+
+  return languages.filter(language => regex.test(language.name));
+};
+
+let app = null;
+
+export const getSuggestionValue = sinon.spy(suggestion => {
+  return suggestion.name;
+});
+
+export const renderSuggestion = sinon.spy(suggestion => {
+  return <span>{suggestion.name}</span>;
+});
+
+export const onChange = sinon.spy((event, { newValue }) => {
+  app.setState({
+    value: newValue
+  });
+});
+
+export const onSuggestionsFetchRequested = sinon.spy(({ value }) => {
+  app.setState({
+    suggestions: getMatchingLanguages(value)
+  });
+});
+
+export default class AutosuggestApp extends Component {
+  constructor() {
+    super();
+
+    app = this;
+
+    this.state = {
+      value: 'C',
```

Comment:
If you set this value here, you don't need to call `setInputValue('C')` too in each test, right?